news/migrations/0017_ratelimit_exempt_group.py

```
@@ -0,0 +1,47 @@
+def delete_ratelimit_exempt_group(apps, schema_editor):
+    """Remove the group; the permission goes with the model."""
+    Group = apps.get_model("auth", "Group")
+    Group.objects.filter(name=RATELIMIT_EXEMPT_GROUP).delete()
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("news", "0016_aidescriptionsettings_descriptiongenerationattempt"),
+        ("auth", "0012_alter_user_first_name_max_length"),
+        ("contenttypes", "0002_remove_content_type_name"),
+    ]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)